*** CMakeLists.txt	2016-05-03 10:50:06.409077384 +0100
--- CMakeLists_VTKPATCH.txt	2016-05-03 10:54:52.480120803 +0100
***************
*** 200,214 ****
  	endforeach()
  endif()
  
- #Locate VTK
- if (Chaste_USE_VTK)
-     #set(VTK_FIND_QUIETLY ON)
-     find_package(VTK REQUIRED)
-     list(APPEND Chaste_INCLUDES "${VTK_INCLUDE_DIRS}")
-     list(APPEND Chaste_LINK_LIBRARIES "${VTK_LIBRARIES}")
-     add_definitions(-DCHASTE_VTK)
- endif()
- 
  #find Boost
  add_definitions( -DBOOST_ALL_NO_LIB )
  set( Boost_USE_STATIC_RUNTIME ON) 
--- 200,205 ----
***************
*** 323,328 ****
--- 314,328 ----
      add_definitions(-DCHASTE_XERCES)
  endif()
  
+ #Locate VTK
+ if (Chaste_USE_VTK)
+     #set(VTK_FIND_QUIETLY ON)
+     find_package(VTK REQUIRED)
+     list(APPEND Chaste_INCLUDES "${VTK_INCLUDE_DIRS}")
+     list(APPEND Chaste_LINK_LIBRARIES "${VTK_LIBRARIES}")
+     add_definitions(-DCHASTE_VTK)
+ endif()
+ 
  add_definitions(-DTRILIBRARY -DTETLIBRARY -DANSI_DECLARATORS)
  
  set(CXXTEST_INCLUDES "${Chaste_BINARY_DIR}/cxxtest")
